hardhat-scripts/deploy-strategy.ts

```
@@ -0,0 +1,186 @@
+      throw new Error(`Invalid address: ${trimmed}`);
+    }
+    return trimmed;
+  });
+}
+
+function stringToUintArray(input: string): number[] {
+  if (!input) return [];
+  return input.split(",").map((num) => {
+    const trimmed = num.trim();
+    const parsed = parseInt(trimmed);
+    if (isNaN(parsed) || parsed.toString() !== trimmed) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)